doc/nrf/known_issues.rst

```diff
@@ -203,6 +203,14 @@ Other issues
 
 .. rst-class:: v2-3-0
 
+NCSDK-20864: TF-M unaligned partitions when MCUBOOT padding and debug optimizations are enabled. 
+  When building TF-M using partition manager with :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` and  :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS` are enabled the resulted partitions will not be aligned with :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS`
+  as required by :ref:`ug_tfm_partition_alignment_requirements` which will cause the build to fail.
```

Comment:
```suggestion
  When building TF-M using the partition manager with options :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` and :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS` enabled, the resulting partitions are not aligned with :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS` as according to :ref:`ug_tfm_partition_alignment_requirements`.
  This will cause the build to fail.
```

Reply:
We are not mentioning :kconfig:option:`CONFIG_DEBUG_OPTIMIZATIONS` in :ref:`ug_tfm_partition_alignment_requirements`... We should.

Reply:
That' my mistake actually, the second Kconfig should be the SPU flash size as mentioned in the alignment requirements. I will update this.